web-animations-1/Overview.bs

```diff
@@ -4412,10 +4421,8 @@ apart from the timing model.
         {{AnimationEffect/getComputedTiming()}} must return a number
         corresponding to the calculated value of the <a>iteration duration</a>
         as defined in the description of the {{EffectTiming/duration}} member of
-        the {{EffectTiming}} interface.
-
-        In this level of the specification, that simply means that an
-        <code>auto</code> value is replaced by zero.
+        the {{EffectTiming}} interface. A percentage value is represented as its
+        double value (i.e. 100% returns 1.0).
```

Comment:
Doesn't this lose information? i.e. it doesn't roundtrip and it's not possible to tell whether 1.0 represents a percent or a millisecond time?

Reply:
It's expected that getComputedTiming can lose information though, right? I thought we had settled on the progress being a double in the [0-1] range in our previous discussions.